index.js

```diff
@@ -35,14 +35,26 @@ module.exports = function (babel) {
     return result;
   }
 
-  function parseScopeObject(buildError, name, node) {
-    if (node.type !== 'ObjectExpression') {
+  function parseScope(buildError, name, node) {
+    let body;
+
+    if (node.type === 'ArrowFunctionExpression') {
+      body = node.body;
```

Comment:
We should handle both types of return within arrow functions. This is a somewhat reasonable authoring format, we shouldn't allow arrow functions of one type but not the other.

Reply:
I wanted to lock it down specifically to an object expression because it reduces complexity, we don't have to actually parse an entire body, verify that said body does not have anything besides a return statement, etc. Just seemed simpler overall.